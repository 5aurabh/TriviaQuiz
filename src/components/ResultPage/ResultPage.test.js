import React from 'react';
import renderer from 'react-test-renderer';
import Resultpage from '.';

it('Resultpage: default', () => {
  const component = renderer.create(
    <Resultpage questions={[]} submittedAnswers={[]} />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
