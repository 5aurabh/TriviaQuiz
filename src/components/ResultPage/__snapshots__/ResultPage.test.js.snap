// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resultpage: default 1`] = `
<div
  className={undefined}
>
  <div
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    <button
      className="undefined "
      disabled={true}
      onClick={undefined}
      style={
        Object {
          "backgroundColor": "tomato",
          "color": "#fff",
          "cursor": "pointer",
        }
      }
    >
      <div
        className="css-na1r9x"
      />
      <span
        style={
          Object {
            "marginLeft": "5px",
            "position": "relative",
            "top": "-5px",
          }
        }
      >
        calculating
      </span>
    </button>
  </div>
</div>
`;
